Project.toml

```diff
@@ -16,7 +16,7 @@ TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
-GitForge = "0.2.3"
+GitForge = "0.2.3, 0.4"
```

Comment:
```suggestion
GitForge = "0.2.3, 0.3, 0.4"
```

Reply:
Also can't comment on the line but needs a project version bump

Reply:
Bumped the version and added `0.3` back in.